docs/how-to/configure-app.md

```diff
@@ -1,18 +1,10 @@
-# Configure the application
+# Configure the Application
 
-By default, configuration options are ingested from pydantic BaseModels,
-however the option exists to override these by defining a yaml file which
-can be passed to the `blueapi` command.
+Blueapi's default configuration can be overridden 
+by defining a yaml file which can be passed to the `blueapi` command.
```

Comment:
memo: this is about to get changed by changes to overriding config values from env vars etc, make sure to update it then